atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -215,14 +184,87 @@ def _send_mail(self, receivers_list, subject, body):
 
         s = None
         try:
-            s = smtplib.SMTP(self.smtp_uri)
+            s = smtplib.SMTP(self.smtp_host)
             s.sendmail(self.from_address, receivers_list, msg.as_string())
```

Comment:
Oh, right. `receivers_list` is being joined into a string for `msg['To']` and the list is passed in `smtplib.SMTP.sendmail`. Not sure why we should pass this list twice to make it work